### PR TITLE
openvino predict component and pipeline example

### PR DESCRIPTION
**Pull request by dtrawins**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/180)

### CODE REVIEW

**Comment by vicaire** [comment]
Hi @dtrawins, 

The comments from one of the robots above provide details instructions on how to pass the CLA check. 

See the comment starting with: 

"Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA)."

Thank you!

**Comment by dtrawins** [comment]
I should be now in the required CLA group. Could you revalidate it now?

**Comment by dtrawins** [comment]
component path changed to contrib/components/openvino as requested

File: samples/openvino/predict/numpy_predict.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import kfp.dsl as dsl
+
```
Comment (vicaire) [inline samples/openvino/predict/numpy_predict.py]: This file is still in the /samples directory. Could we move it to the /contrib directory?

File: samples/openvino/predict/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# OpenVINO predict pipeline
+
+This is an example of simple one step pipeline implementation including OpenVINO predict component.
```
Comment (vicaire) [inline samples/openvino/predict/README.md]: This file is still in the /samples directory. Could we move it to the /contrib directory?

Comment (dtrawins) [reply samples/openvino/predict/README.md]: samples are now also moved to contrib

File: samples/openvino/predict/numpy_predict.py (hunk 1 above)
Comment (dtrawins) [reply samples/openvino/predict/numpy_predict.py]: samples are now also moved to contrib

**Comment by vicaire** [comment]
Bypassing the e2e tests since they are not affected by these changes.

**Pull request merged**